src/makeCalculator.js

```diff
@@ -5,6 +5,28 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+  };
+
+  calculator.reset = () => {
+    calculator.result = 0;
+
+    return calculator;
+  };
+
+  calculator.add = (number) => (calculator.result += number);
+  calculator.subtract = (number) => (calculator.result -= number);
+  calculator.multiply = (number) => (calculator.result *= number);
+  calculator.divide = (number) => (calculator.result /= number);
```

Comment:
The `divide` method should handle division by zero to prevent runtime errors. Consider adding a check to ensure `number` is not zero before performing the division.